kubetest/anywhere.go

```diff
@@ -168,7 +168,10 @@ func (k kubernetesAnywhere) IsUp() error {
 }
 
 func (k kubernetesAnywhere) DumpClusterLogs(localPath, gcsPath string) error {
-	return defaultDumpClusterLogs(localPath, gcsPath)
+	// TODO: the default implementation (log-dump.sh) doesn't work for
+	// kubernetes-anywhere yet.
+	// https://github.com/kubernetes/kubeadm/issues/256
```

Comment:
maybe add something like `log.Print("kubernetesAnythere logdump - Noop")` so that later we can still see where the logdump should come from